sw360chores.pl

```diff
@@ -195,11 +202,30 @@ =head1 SYNOPSIS
         if ($prod) {
             push(@toCall, "-f", "deployment/docker-compose.prod.yml");
         }else{
+            $ENV{'DEVELOPMENT_MODE'} = 1;
+
             mkdir "_deploy" if ! -d "_deploy";
             push(@toCall, "-f", "deployment/docker-compose.dev.yml");
             if($cveSearch) {
                 push(@toCall, "-f", "deployment/docker-compose.dev.cve-search-server.yml");
             }
+
+            if($quickDeploy) {
+                $ENV{'QUICK_DEPLOY'} = 1;
+
+                if(!defined $ENV{'SW360_REPO_PATH'}) {
+                    say "SW360_REPO_PATH non set. Defaulting to [../sw360].";
+                    $ENV{'SW360_REPO_PATH'} = realpath(getcwd() . '/../sw360');
+                }
+                $ENV{'PORTLET_WEBAPP_SRC'} = $ENV{'SW360_REPO_PATH'} . "/frontend/sw360-portlet/src/main/webapp";
+                say "===>";
+                say "===> Quick Deploy will be activated. Portlet sources are loaded from [" . $ENV{'PORTLET_WEBAPP_SRC'} . "].";
+                say "===>";
+            } else {
+                # Use dummy dir for volume because we cannot disable the volume by condition in the docker compose file
```

Comment:
but one can conditionally load a yml file `deployment/docker-compose.dev.quick-deploy.yml` which only adds the volume if needed.